addons/web/static/src/core/attachment_viewer/attachment_viewer.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
 
-    <t t-name="mail.AttachmentViewer" owl="1">
+    <t t-name="web.AttachmentViewer" owl="1">
         <div class="d-flex justify-content-center" t-att-class="{ 'modal fixed-top bottom-0': props.modal }">
-            <div class="o-mail-AttachmentViewer flex-column align-items-center d-flex w-100 h-100" tabindex="0">
-                <div class="o-mail-AttachmentViewer-header position-absolute top-0 d-flex w-100 bg-black-75 text-400">
+            <div class="o-web-AttachmentViewer flex-column align-items-center d-flex w-100 h-100" tabindex="0">
```

Comment:
I believe components in the `web` module have the privilege to not require the `web` prefix.